ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques.csl

```diff
@@ -0,0 +1,622 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>École Pratique des Hautes Études - Sciences historiques et philologiques (French)</title>
+    <title-short>EPHE-SHP</title-short>
+    <id>http://www.zotero.org/styles/ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques</id>
+    <link rel="self" href="http://www.zotero.org/styles/ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques"/>
+    <link href="http://www.zotero.org/styles/invisu" rel="template"/>
+    <link href="http://www.zotero.org/styles/le-tapuscrit-note" rel="template"/>
+    <link href="http://www.zotero.org/styles/traces" rel="template"/>
+    <author>
+      <name>Anne-Sophie Bessero</name>
+    </author>
+    <author>
+      <name>Laurent Garrigues</name>
+    </author>
+    <category citation-format="author"/>
+    <category citation-format="note"/>
```

Comment:
First it was "note" but your engine TRAVIS asked to replace by 'in-text"…
I will try again with "note".

> Le 6 avr. 2016 à 19:42, Rintze M. Zelle <notifications@github.com> a écrit :
> 
> In ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques.csl <https://github.com/citation-style-language/styles/pull/2006#discussion_r58749268>:
> 
> > +  <info>
> > +    <title>École Pratique des Hautes Études - Sciences historiques et philologiques (French)</title>
> > +    <title-short>EPHE-SHP</title-short>
> > +    <id>http://www.zotero.org/styles/ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques</id>
> > +    <link rel="self" href="http://www.zotero.org/styles/ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques"/>
> > +    <link href="http://www.zotero.org/styles/invisu" rel="template"/>
> > +    <link href="http://www.zotero.org/styles/le-tapuscrit-note" rel="template"/>
> > +    <link href="http://www.zotero.org/styles/traces" rel="template"/>
> > +    <author>
> > +      <name>Anne-Sophie Bessero</name>
> > +    </author>
> > +    <author>
> > +      <name>Laurent Garrigues</name>
> > +    </author>
> > +    <category citation-format="author"/>
> > +    <category citation-format="note"/>
> A style can have only a single "citation-format" (see http://docs.citationstyles.org/en/stable/specification.html#info <http://docs.citationstyles.org/en/stable/specification.html#info>).
> 
> This looks like a "note" style, but you're using <style class="in-text" ... > above, so I'm a little confused. Does this style use in-text citations (value “in-text”) or footnotes/endnotes (“note”)?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/citation-style-language/styles/pull/2006/files/d9c47a3300573e584ccc3afe538467ffe00db441#r58749268>